core/src/main/scala/tech/beshu/ror/boot/ReadonlyRest.scala

```diff
@@ -172,6 +173,9 @@ trait ReadonlyRest extends Logging {
               httpClientsFactory = httpClientsFactory,
               ldapConnectionPoolProvider
             )
+            if (engine.context.involvesFields) {
+              logger.warn("ACL contains fields rule. To make it work well, all nodes should have ROR plugin installed.")
```

Comment:
WDYT about improving it? this warn makes no sense when fields rule is used in proxy mode or plugin mode with ES FLS engine